### PR TITLE
Update 2020.paclic.xml

### PR DESCRIPTION
**Pull request by huyen144**
Metadata changes: 

Posted issue:
#1313 
2020.PACLIC-1.3 
Current title in metadata:  "A new look a99t Pattani Malay Initial Geminates: a statistical and machine learning approach"
Changed to:  "A new look at Pattani Malay Initial Geminates: a statistical and machine learning approach"

7 new corrections (due to different paper titles in EasyChair metadata and accent on author name):

2020.PACLIC-1.23
Current author name: "Elisabeth Metais"
Changed to: "Elisabeth Métais"

2020.PACLIC-1.32
Current title in metadata: "Evaluation of BERT Models by Using Sentence Clustering"
Changed to: "Evaluation of Pretrained BERT Model by Using Sentence Clustering"

2020.PACLIC-1.44
Current title in metadata: "Research on Prosody of Collaborative Construction in Mandarin Conversation"
Changed to: "Prosody Features of Collaborative Construction in Mandarin Conversation"

2020.PACLIC-1.47
Current title in metadata: "Exploring Discourse of Same-sex Marriage in Taiwan: A Case Study of Near-Synonym of HOMOSEXUAL in Opposing Stances"
Changed to: "Exploring Discourse on Same-sex Marriage in Taiwan: A Case Study of Near-Synonym of HOMOSEXUAL in Opposing Stances"

2020.PACLIC-1.48
Current title in metadata: "A simple and efficient ensemble classifier combining multiple neural network models on social media datasets in Vietnamese"	 
Changed to: "A Simple and Efficient Ensemble Classifier Combining Multiple Neural Network Models on Social Media Datasets in Vietnamese"

2020.PACLIC-1.54
Current title in metadata: "Attention-based Domain adaption Using Transfer Learning for Part-of-Speech Tagging: An Experiment on the Hindi language"
Changed to: "Attention-based Domain Adaption Using Transfer Learning for Part-of-Speech Tagging: An Experiment on the Hindi Language"

2020.PACLIC-1.59
Current title in metadata: "Redefining verbal nouns in Japanese: From the perspective of polycategoriality"
Changed to: "Redefining Verbal Nouns in Japanese: From the Perspective of Polycategoriality"

### CODE REVIEW

File: data/xml/2020.paclic.xml [hunk 1]
```diff
@@ -399,14 +399,14 @@
...
-      <title>Exploring Discourse of Same-sex Marriage in <fixed-case>T</fixed-case>aiwan: A Case Study of Near-Synonym of <fixed-case>HOMOSEXUAL</fixed-case> in Opposing Stances</title>
+      <title>Exploring Discourse on Same-sex Marriage in <fixed-case>T</fixed-case>aiwan: A Case Study of Near-Synonym of <fixed-case>HOMOSEXUAL</fixed-case> in Opposing Stances</title>
       <author><first>Han-Tang</first><last>Hung</last></author>
       <author><first>Shu-Kai</first><last>Hsieh</last></author>
       <pages>411–419</pages>
       <url hash="549ab4b1">2020.paclic-1.47</url>
     </paper>
     <paper id="48">
-      <title>A simple and efficient ensemble classifier combining multiple neural network models on social media datasets in <fixed-case>V</fixed-case>ietnamese</title>
+      <titleA Simple and Efficient Ensemble Classifier Combining Multiple Neural Network Models on Social Media Datasets in <fixed-case>V</fixed-case>ietnamese</title>
```
Comment (mjpost) [inline data/xml/2020.paclic.xml:409]: You've introduced an error here that broke the build.

Comment (huyen144) [reply data/xml/2020.paclic.xml:409]: Oh I am sorry, I am correcting that.

**Review by mjpost — APPROVED** [review]
LGTM.

**Comment by huyen144** [comment]
Thank you!

**Pull request merged**